packages/web/src/components/dashboard/volume-chart-info/VolumeChartInfo.styles.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import mixins from "@styles/mixins";
+import { fonts } from "@constants/font.constant";
+
+export const VolumeChartInfoWrapper = styled.div`
+  ${mixins.flexbox("column", "flex-start", "flex-start")};
+  background-color: ${({ theme }) => theme.color.background01};
+  border-radius: 8px;
+  padding: 24px 24px 23px 24px;
+  gap: 16px;
+  width: 620px;
```

Comment:
The chart area doesn't seem to be flexible.

Reply:
I'll add. thank you.